pkg/kubelet/qos/policy_test.go

```diff
@@ -229,6 +264,24 @@ func TestGetContainerOOMScoreAdjust(t *testing.T) {
 			lowOOMScoreAdj:  -997,
 			highOOMScoreAdj: -997,
 		},
+		{ // low and high range differ from burstableMultiContainerMediumPod & burstableMultiContainerLargePod
```

Comment:
I think it would be better to have test cases that include regular Init Containers and restartable Init Containers.

Reply:
The current and previous logic do not account for init containers **in the calculation.** Do you think it would be a good idea to include init containers in the tests?

Reply:
Since you modified it to include `PodRequests()` in the calculations, doesn’t this mean that Regular / Restartable Init Containers also now affect the calculations?

Reply:
Isn't `GetContainerOOMScoreAdjust()` originally supposed to be executed for Init Containers and also for Ephemeral Containers as well?

Reply:
I agree, I added some tests related to these cases.